jwst/cube_build/file_table.py

```diff
@@ -120,7 +118,8 @@ def set_file_table(self,
                 else:
 
                     log.info('Instrument not valid for cube')
-        return instrument, detector
+#        return instrument, detector
```

Comment:
Any reason to leave this here? Unless there is a significant reason that is documented, its better practice to just delete such lines